src/lib/Runner.ts

```diff
@@ -50,9 +50,16 @@ export class Runner {
         landRequest: landRequest.get(),
       });
 
+      const triggererUserMode = await permissionService.getPermissionForUser(
```

Comment:
hm I found the name of the function a bit weird - I tot understand but this is to get the permissionLevel right?

Reply:
The terms 'user mode' and 'permission level' seem to be used interchangeably throughout the app, I can change it to make it clearer though.

Reply:
it might make it clearer